cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -42,15 +50,20 @@ type InitConfiguration struct {
 
 	// NodeRegistration holds fields that relate to registering the new master node to the cluster
 	NodeRegistration NodeRegistrationOptions
+}
+
+func (ic InitConfiguration) Fuzz(c fuzz.Continue) {}
```

Comment:
It could be useful to add a comment explaining why this line is necessary

Reply:
removed